Source/ACE/Entity/Landblock.cs

```diff
@@ -87,8 +85,8 @@ public Landblock(LandblockId id)
             var creatures = DatabaseManager.World.GetCreaturesByLandblock(this.id.Landblock);
             foreach (var c in creatures)
             {
-                Creature cwo = new Creature(c);
-                worldObjects.Add(cwo.Guid, cwo);
+                // Creature cwo = new Creature(c);
+                // worldObjects.Add(cwo.Guid, cwo);

```

Comment:
You disabled creature spawning here. Is there another place where this is happening now, as I couldn't really find it anywhere else.